src/poissontime.jl

```diff
@@ -29,6 +29,42 @@ function poisson_time(a, b, u)
     end
 end
 
+"""
+    poisson_time((a, b, c), u)
+
+Obtaining waiting time for inhomogeneous Poisson Process
+with rate of the form λ(t) = c + (a + b*t)^+, 
+where `c`> 0 ,`a, b` ∈ R, `u` uniform random variable
+"""
+function poisson_time((a,b,c)::NTuple{3}, u)
+    if b > 0
+        if a < 0
+            if !(-c*a/b + log(u) >= 0)
```

Comment:
```suggestion
            if -c*a/b + log(u) <= 0.0
```

Reply:
`<` or `<=`?

Reply:
Maybe better `<` (but it should be equivalent)

Reply:
You can edit your comment to change it

Reply:
I could